frontend/src/Frontend/UI/Dialogs/Send.hs

```diff
@@ -542,15 +523,28 @@ finishCrossChainTransfer netInfo fromIndex fromAccount ucct toGasPayer = Workflo
       toChain = _unfinishedCrossChainTransfer_recipientChain ucct
       requestKey = _unfinishedCrossChainTransfer_requestKey ucct
   resultOk <- divClass "modal__main" $ do
-    divClass "title" $ text "Transaction Status"
-    divClass "group" $ do
+    elClass "h2" "heading heading_type_h2" $ text "Transaction Status"
+
+    (resultOk0, errMsg, retry) <- divClass "group" $ do
       elClass "ol" "transaction_status" $ do
         let item ds = elDynAttr "li" (ffor ds $ \s -> "class" =: statusText s)
-        item (pure Status_Done) $ do
-          el "p" $ text $ "Cross chain transfer initiated on chain " <> _chainId fromChain
-          el "p" $ crossChainMessage $ "Request key: " <> Pact.requestKeyToB16Text requestKey
+        item (pure Status_Done) $ el "p" $
+          text $ "Cross chain transfer initiated on chain " <> _chainId fromChain
         pb <- getPostBuild
         runUnfinishedCrossChainTransfer netInfo fromChain toChain toGasPayer $ requestKey <$ pb
+
+    elClass "h2" "heading heading_type_h2" $ text "Request Key"
+    divClass "group" $ text $ Pact.requestKeyToB16Text requestKey
+
+    void $ runWithReplace blank $ ffor (leftmost [Just <$> errMsg, Nothing <$ retry]) $ \case
```

Comment:
Maybe use `widgetHold_` since we're not consuming anything from `runWithReplace` anyway. 
Could also do `$ traverse $ \e ->`  instead of `\case` depending on your preferences